### PR TITLE
Channel

### PR DESCRIPTION
**Pull request by ArchiComputer**
## ✍️ Describe your changes
creating channel works in FAST API docs

## 🎟 ️Issue ticket number and link
closes # 

## ✅ Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: app/schemas/account.py [hunk 1]
```diff
@@ -24,6 +32,21 @@ class AccountCreateResponse(AccountBase):
     pass
 
 
+class UserCreate(UserBase):
+    pass
+
+
+class User(UserBase):  # UserBase를 상속받음
```
Comment (yhware) [inline app/schemas/account.py:39]: Please separate user related schemas to a dedicated file?

File: app/services/channel.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from sqlalchemy.sql import func
+from app.models.channel import Channel
+
+
+async def create_channel(dto, db):
+    new_channel = Channel(
+        application_id=dto.application_id,
+        name=dto.name,
+        # TODO: should discuss Daniel what this Max_members value should be?
+        max_members=20,
```
Comment (yhware) [inline app/services/channel.py:10]: let's set the max members to a static value of 100 for now.

File: app/services/channel.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        created_at=func.now(),
```
Comment (yhware) [inline app/services/channel.py:11]: remove assignment of created_at and updated_at

**Review by yhware — CHANGES_REQUESTED** [review]

File: app/schemas/account.py (hunk 1 above)
Comment (ArchiComputer) [reply app/schemas/account.py:39]: I moved it to schemas/user.py

File: app/services/channel.py (hunk 2 above)
Comment (ArchiComputer) [reply app/services/channel.py:10]: ok. I will leave it for Max to set its number

**Review by yhware — APPROVED** [review]

**Pull request merged**